README.md

```diff
@@ -25,6 +25,7 @@ serviceDescription: |                   # A description of the service
   Use this paragraph to describe your service. It can go over multiple lines
   like this.
 serviceDomain: www.tax.service.gov.uk   # The domain name under which this service exists (exclude the https:// and the path)
+mobilePlatform:                         # android|ios (optional)
```

Comment:
I would maybe consider adding additional comment here, e.g.:
`android|ios (optional) - Only add this section if your statement is for a mobile app`